example/opencv_histogram.cpp

```diff
@@ -11,7 +11,9 @@
 //Code sample for calculating histogram using OpenCL and
 //displaying image histogram in OpenCV.
 
-#define BOOST_COMPUTE_DEBUG_KERNEL_COMPILATION
+#ifndef BOOST_COMPUTE_DEBUG_KERNEL_COMPILATION
+    #define BOOST_COMPUTE_DEBUG_KERNEL_COMPILATION
+#endif
```

Comment:
We should probably just remove this define from the file. It should only really be defined by the user when they are debugging.